src/emqx_sn_gateway.erl

```diff
@@ -771,9 +784,10 @@ send_connack(State) ->
 
 send_message(Msg = #mqtt_sn_message{type = Type},
              State = #state{sockpid = SockPid, peername = Peername}) ->
-    ?LOG(debug, "SEND ~s~n", [emqx_sn_frame:format(Msg)]),
+    ?LOG(info, "SEND ~s~n", [emqx_sn_frame:format(Msg)]),
     inc_outgoing_stats(Type),
     Data = emqx_sn_frame:serialize(Msg),
+    ?LOG(debug, "SEND ~0p", [Data]),
```

Comment:
should we really log twice for the same message ?

Reply:
Does this look better? Under the info level, it is possible to print the content of the message in a human-readable way. Printing out binary data messages again at debug level would help diagnose problems with message serialization?

Reply:
I’d argue both should be debug

Reply:
fixed